playbooks/group_vars/all/global.yml

```diff
@@ -14,7 +14,7 @@ force_minishift_install: false
 force_repo_clone: false
 
 # Use General DNS in case of VPN DNS failures
-dns_server: 8.8.8.8
+#dns_server: 8.8.8.8
```

Comment:
Since removing the comment won't do anything since the switch is removed from the playbook.  Do you think we could just remove it?

Reply:
Can we just remove the line altogether.  This is just cruft.